AlphaWallet/Core/Types/RealmStore.swift

```diff
@@ -24,9 +25,9 @@ final class RealmStore {
             callback(realm)
         } else {
             dispatchPrecondition(condition: .notOnQueue(queue))
-            queue.sync { [unowned self] in
+            queue.sync {
                 autoreleasepool {
-                    let realm = try! Realm(configuration: config)
```

Comment:
Why does the `try! Realm(configuration:)` call fail sometimes?

Reply:
its not, just to be sure